src/wallet/wallet.cpp

```diff
@@ -1300,18 +1268,7 @@ CAmount CWallet::GetDebit(const CTxIn &txin, const isminefilter &filter) const {
 isminetype CWallet::IsMine(const CTxOut &txout) const {
     LOCK(cs_wallet);
 
-    if (txout.scriptPubKey.IsZerocoinMint()) {
-        CWalletDB db(strWalletFile);
-        CBigNum pub;
-
-        try {
-            pub = ParseZerocoinMintScript(txout.scriptPubKey);
-        } catch (std::invalid_argument&) {
-            return ISMINE_NO;
-        }
-
-        return db.HasZerocoinEntry(pub) ? ISMINE_SPENDABLE : ISMINE_NO;
-    } else if (txout.scriptPubKey.IsSigmaMint()) {
+    if (txout.scriptPubKey.IsSigmaMint()) {
```

Comment:
This one will cover zerocoin to sigma remints too. Need to test this case to see if nothing breaks